tools/local-network/authorize-minters.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+
+# This script can be run after starting a local network in order to generate and
+# authorize minters.
+#
+# It must be kept in sync with the governor setup in local_network.py
+#
+# This is a way to prepare the network for running the mint auditor integration tests
```

Comment:
@eranrund lmk how you feel about this being a separate script

It might also be okay to just add this to the local network script, but I thought (1) maybe there's a reason it wasn't added before (2) it's less changes over all if it's just a separate thing

Reply:
There's no real reason why it wasn't added. I think it could be nice if the local network script already did it, but it is less changes and that script is already long and probably is nearing the point of justifying breaking it into multiple files. I don't have a strong feeling about this being its own script as long as it continues to be used in CI since that will prevent it from rotting.